packages/core/stories/progress/progress.qa.stories.tsx

```diff
@@ -35,6 +35,12 @@ export const ExamplesGrid: StoryFn<QAContainerProps> = (props) => {
         hideLabel
       />
       <CircularProgress aria-label="Download" value={38} hideLabel />
+      <LinearProgress
+        aria-label="Download"
+        className={className}
+        style={{ padding: "50px" }}
+        variant="indeterminate"
```

Comment:
Will animation makes QA story always changing? Any possibility to override certain CSS to make it stable?

Reply:
Good point, I'll look at what we did for spinner